src/plugins/chart_expressions/expression_metric/public/components/with_auto_scale.styles.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { css } from '@emotion/react';
+
+export const autoScaleWrapperStyle = css({
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  maxWidth: '100%',
+  maxHeight: '100%',
+  overflow: 'hidden',
+  lineHeight: 1.5,
```

Comment:
Is it possible to use the `euiLineHeight` EUI variable here?

Reply:
done